osquery/tables/system/windows/drivers.cpp

```diff
@@ -110,37 +119,37 @@ Status getDeviceProperty(const device_infoset_t& infoset,
                   "Error getting buffer size for device property");
   }
 
-  std::unique_ptr<BYTE, decltype(freePtr)> drvBuff(
-      static_cast<PBYTE>(malloc(buffSize)), freePtr);
-  if (drvBuff == nullptr) {
+  std::unique_ptr<BYTE, decltype(kFreePtr)> drv_buff(
+      static_cast<PBYTE>(malloc(buff_size)), kFreePtr);
+  if (drv_buff == nullptr) {
     return Status(1, "Failed to malloc for device property buffer");
   }
 
   ret = SetupDiGetDevicePropertyW(infoset.get(),
                                   &device,
                                   &prop,
-                                  &devPropType,
-                                  drvBuff.get(),
-                                  buffSize,
+                                  &dev_prop_type,
+                                  drv_buff.get(),
+                                  buff_size,
                                   nullptr,
                                   0);
   if (ret == FALSE) {
     return Status(GetLastError(), "Error getting device property");
   }
 
-  if (devPropType == DEVPROP_TYPE_UINT32) {
-    result = std::to_string(*(PUINT32)drvBuff.get());
-  } else if (devPropType == DEVPROP_TYPE_INT32) {
-    result = std::to_string(*(PINT32)drvBuff.get());
-  } else if (devPropType == DEVPROP_TYPE_STRING) {
-    std::wstring name((PWCHAR)drvBuff.get());
+  if (dev_prop_type == DEVPROP_TYPE_UINT32) {
+    result = std::to_string(*(PUINT32)drv_buff.get());
+  } else if (dev_prop_type == DEVPROP_TYPE_INT32) {
+    result = std::to_string(*(PINT32)drv_buff.get());
+  } else if (dev_prop_type == DEVPROP_TYPE_STRING) {
+    std::wstring name((PWCHAR)drv_buff.get());
     result = std::string(name.begin(), name.end());
-  } else if (devPropType == DEVPROP_TYPE_FILETIME) {
+  } else if (dev_prop_type == DEVPROP_TYPE_FILETIME) {
     result =
-        std::to_string(osquery::filetimeToUnixtime(*(PFILETIME)drvBuff.get()));
+        std::to_string(osquery::filetimeToUnixtime(*(PFILETIME)drv_buff.get()));
   } else {
     return Status(
-        1, "Unhandled device property type " + std::to_string(devPropType));
+        1, "Unhandled device property type " + std::to_string(dev_prop_type));
```

Comment:
NIT Status::failure

Reply:
There's quite a few of them actually, I changed the rest to be `::failure` and include the error code in the message, which I think makes a better standard for printing Windows error codes.